docs/snowflake-native-apps/linkedin-ads.md

```diff
@@ -0,0 +1,109 @@
+The Airbyte connector for LinkedIn Ads is a Snowflake Native Application that allows you to extract data from your LinkedIn Ads account and load records into a database of your choice within Snowflake.
+
+> **Info**  
+> The LinkedIn Ads Connector is in _private preview_ and is subject to further development that may affect setup and configuration of the application.
+
+# Getting started
+
+## Prerequisite
+A LinkedIn Ads account with permission to access data from accounts you want to sync.
+
+## Installing the App
+
+> **Warning**  
+> Do not refresh the Apps page while the application is being installed. This may cause installation to fail.
+
+1. Log into your Snowflake account.
+2. On the left sidebar, click Marketplace.
+3. Search for `Linkedin Ads Connector` by Airbyte or navigate to https://app.snowflake.com/marketplace/listing/GZTYZ9BCRSJ/airbyte-linkedin-ads-connector-by-airbyte
+4. Click `Request`. This will send a request that we will manually service as soon as we can.
+5. On the left sidebar, click `Apps.
+6. Under the `Recently Shared with You` section, you should see the `Linkedin Ads Connector by Airbyte`. Click `Get`.
+7. Expand `Options`.
+    1. You can rename the application or leave the default. This is how you will reference the application from a worksheet
+    2. Specify the warehouse that the application will be installed to.
+8. Click `Get`.
+9. Wait for the application to install. Once complete, the pop-up window should automatically close.
+
+You should now see the application under `Installed Apps`. You may need to refresh the page.
+
+## LinkedIn Account
+In order for the Snowflake Native App to query LinkedIn, you will need an account with the right permissions. Please follow [the LinkedIn Ads authentication guide](https://docs.airbyte.com/integrations/sources/linkedin-ads/#set-up-linkedin-ads-authentication-airbyte-open-source) for further information.
+
+## Snowflake Native App Authorizations
+> **Note**  
+> All the queries below use `LINKEDIN_ADS_TEST` but if you have renamed your application, you will have to use this name as a reference.
```

Comment:
I added this line based on [this line](https://github.com/airbytehq/airbyte/pull/30386/files#diff-9199f80dfa9350b869d60d63787fea386c7146e37b9600d30961d174066235c2R23) but I haven't tested this

Reply:
i think we should do some sort of bracketing like `<your db>` or something, but LINKEDIN_ADS_TEST is not too nice to see, i'll update relevant parts